src/layouts/index.css

```diff
@@ -39,7 +31,7 @@ p:last-child {
   margin: 1px;
   height: 8px;
   width: 8px;
-  background: rgb(251, 127, 90);
+  background: #ef891c
```

Comment:
Die Datei sollten wir dann bald auch noch in Styled Components umwandeln. Hab ich anscheinend übersehen beim refactorn 🙈

Reply:
jap hast du :D das dachte ich mir auch